app/controllers/orders/new.js

```diff
@@ -10,14 +10,16 @@ export default Controller.extend({
         for (const attendee of attendees ? attendees.toArray() : []) {
           await attendee.save();
         }
-        if (paymentMode === 'free' || paymentMode === 'bank' || paymentMode === 'cheque' || paymentMode === 'onsite') {
+        if (paymentMode === 'free') {
           order.set('status', 'completed');
-        } else {
+        } else if (paymentMode === 'bank' || paymentMode === 'cheque' || paymentMode === 'onsite') {
           order.set('status', 'placed');
+        } else {
+          order.set('status', 'pending');
         }
         await order.save()
           .then(order => {
-            if (order.status === 'placed') {
+            if (order.status === 'pending' || order.status === 'placed') {
```

Comment:
@shreyanshdwivedi If the order status is placed, ergo it was an order via bank cheques or something, why would we ask t fill payment details?

Reply:
@CosmicCoder96 I agree. It was present earlier. I'll update it :)